epics/dbr.py

```diff
@@ -356,8 +356,8 @@ class connection_args(ctypes.Structure):
 class access_rights_handler_args(ctypes.Structure):
     "access rights arguments"
     _fields_ = [('chid', chid_t),
-                ('read_access', uint_t, 1),
-                ('write_access', uint_t, 1)]
+                ('read_access', ctypes.c_bool),
```

Comment:
`c_bool` is wrong here, as I believe it's just an alias for `char`. 

That'd mean our previous bitfield which indicated `2 bits of 1 full byte` is now 2 full bytes instead.